src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -58,6 +58,9 @@ internal partial class CurlHandler : HttpMessageHandler
             new KeyValuePair<string,CURLAUTH>("Basic", CURLAUTH.Basic),
         };
 
+        // Max timeout value used by WinHttp handler, so mapping to that here.
+        private static readonly TimeSpan s_maxTimeout = TimeSpan.FromMilliseconds(int.MaxValue);
```

Comment:
Nit: Which is preferred: `readonly static` or `static readonly`? Should it be consistent with the rest of the static readonly fields below?